platformio.ini

```diff
@@ -143,11 +143,16 @@ build_flags = ${common_esp32.build_flags} ${common_wifi.build_flags} ${common_bt
 extends = common_esp32_s3
 lib_deps = ${common.lib_deps}
 
+[env:usb]
+extends = common_esp32_s3
+lib_deps = ${common.lib_deps} ${common.wifi_deps}
+build_flags = ${common_esp32.build_flags}  ${common_wifi.build_flags} -DARDUINO_USB_MODE=1 -DARDUINO_USB_CDC_ON_BOOT=1
+
 [env:wifi_s3]
 extends = common_esp32_s3
 lib_deps = ${common.lib_deps} ${common.wifi_deps}
 build_flags = ${common_esp32.build_flags}  ${common_wifi.build_flags} -DARDUINO_USB_MODE=1 -DARDUINO_USB_CDC_ON_BOOT=1
-upload_protocol = espota
+#upload_protocol = espota
```

Comment:
@BarbourSmith sorry, I didn't mean to comment this! you should put that back in, sorry!

Reply:
Great catch!